eclair-core/src/main/scala/fr/acinq/eclair/channel/Register.scala

```diff
@@ -69,15 +70,19 @@ class Register extends Actor with ActorLogging {
       val compatReplyTo = if (replyTo == ActorRef.noSender) sender() else replyTo
       channels.get(channelId) match {
         case Some(channel) => channel.tell(msg, compatReplyTo)
-        case None => compatReplyTo ! ForwardFailure(fwd)
+        case None =>
+          println(ForwardFailure(fwd))
+          compatReplyTo ! ForwardFailure(fwd)
```

Comment:
nit: revert

Reply:
I added them back on purpose, because the race is still there.